js/script.js

```diff
@@ -17,3 +16,152 @@ navMenu.forEach((links) => links.addEventListener('click', () => {
+    liveLink: '#',
+    sourceLink: '#',
+  }, {
+    name: 'Profesional Art Printing Data More',
+    image: 'assets/mobile/Portfolio2.svg',
+    imageDescription: 'Image Design',
+    description:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industrys standard.',
+    techList: ['HTML', 'Bootstrap', 'Ruby'],
+    liveLink: '#',
+    sourceLink: '#',
+  },
+  {
+    name: 'Data Dashboard Healthcare',
+    image: 'assets/mobile/Portfolio3.svg',
+    imageDescription: 'Portfolio image',
+    description:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industrys standard.',
+    techList: ['HTML', 'Bootstrap', 'Ruby'],
+    liveLink: '#',
+    sourceLink: '#',
+  },
+  {
+    name: 'Website Portfolio',
+    image: 'assets/mobile/Portfolio.svg',
+    imageDescription: 'Portfolio image placeholder',
+    description:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industrys standard.',
+    techList: ['HTML', 'Bootstrap', 'Ruby'],
+    liveLink: '#',
+    sourceLink: '#',
+  },
+  {
+    name: 'Profesional Art Printing Data More',
+    image: 'assets/mobile/Portfolio2.svg',
+    imageDescription: 'Image Design',
+    description:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industrys standard.',
+    techList: ['HTML', 'Bootstrap', 'Ruby'],
+    liveLink: '#',
+    sourceLink: '#',
+  },
+  {
+    name: 'Data Dashboard Healthcare',
+    image: 'assets/mobile/Portfolio3.svg',
+    imageDescription: 'Portfolio image',
+    description:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industrys standard.',
+    techList: ['HTML', 'Bootstrap', 'Ruby'],
+    liveLink: '#',
+    sourceLink: '#',
+  },
+  {
+    name: 'Website Portfolio',
+    image: 'assets/mobile/Portfolio3.svg',
+    imageDescription: 'Portfolio image placeholder',
+    description:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industrys standard.',
+    techList: ['HTML', 'Bootstrap', 'Ruby'],
+    liveLink: '#',
+    sourceLink: '#',
+  },
+];
+const modal = document.querySelector('.modal');
+const modalClose = document.querySelector('.close-button');
+
+modalClose.addEventListener('click', () => {
+  modal.classList.remove('modal-open');
+});
+
+function ShowModal(index) {
+  modal.classList.toggle('modal-open');
+  const modalTitle = document.querySelector('.name');
+  const modalImage = document.querySelector('.modal-image');
+  const projectDescription = document.querySelector('.project-description');
+  const techList = document.querySelector('.tech-container');
+  const seeLive = document.querySelector('.see-live');
+  const seeSource = document.querySelector('.see-source');
+  const project = projectsLists[index];
```

Comment:
- [ ] Kindly work more on the modal container in accordance to the given template. The container supposed to have margin-bottom property. **Note: If there is a need for vertical scroll bar, I think you can add it to the modal container so that all the element would be accessible. Thanks.**

## Your Design
<img width="946" alt="DSC_2740-removebg-preview" src="https://github.com/Kccendana/portfolio_project/assets/103386183/9f757659-d2f0-4234-ba68-b8b25c1591f7">

<img width="280" alt="DSC_2740-removebg-preview" src="https://github.com/Kccendana/portfolio_project/assets/103386183/1793dd5c-859b-48b7-a1b8-da6c9c6e63d6">

## Figma Design
<img width="541" alt="DSC_2740-removebg-preview" src="https://github.com/Kccendana/portfolio_project/assets/103386183/967d15c1-dc2a-49cb-8ce8-0a84af7307ad">

<img width="144" alt="DSC_2740-removebg-preview" src="https://github.com/Kccendana/portfolio_project/assets/103386183/c9ec10c2-d117-4679-bfac-c94b67b7e8de">